### DISCUSSION TITLE
dbt is not installed -- An error occured while initializing the dbt project: cannot import name 'DEFAULT_PROFILES_DIR' from 'dbt.flags'

### DISCUSSION THREAD

**Discussion by treefinger** [AltimateAI/vscode-dbt-power-user#1014] [Q&A] [answered]
Hello, 

I've been struggling to get the extension working. I've got python installed, and dbt installed, proven by `which python` and `which dbt`, I am able to run models as usual in the CLI and my profiles.yml file is being found and used correctly, etc.

No matter what I try I can't seem to get around this message:

```An error occured while initializing the dbt project: cannot import name 'DEFAULT_PROFILES_DIR' from 'dbt.flags' (/opt/homebrew/lib/python3.11/site-packages/dbt/flags.py)```

I've tried changing the interpreter, overriding the profiles dir path via the setting (set to . , and other things) to no avail.

I'm sure it's something obvious, but some help would be appreciated. Thank you!

**Comment by treefinger** [comment]
snippet from `dbt debug` 

![image](https://github.com/AltimateAI/vscode-dbt-power-user/assets/8889070/20d4ebcb-3f4f-4972-ad4a-14903fe1b191)

**Comment by mdesmet** [comment, marked as answer]
This issue has been fixed a long time already in the extension (basically it is support for a very old dbt version). You are probably on a very old version of vscode that doesn't seem to self update. Please update or reinstall vscode and you should be able to update to the latest version of the extension.
